src/atoms/TextArea/index.jsx

```diff
@@ -0,0 +1,36 @@
+export const TextArea = (props) => {
+  const {
+    className,
```

Comment:
Can you please add your new `maxChar` prop here to the list of props, as well as in the props validation below :)